sdk/search/Azure.ResourceManager.Search/src/Customization/ArmSearchModelFactory.cs

```diff
@@ -47,7 +47,7 @@ public static SearchServiceData SearchServiceData(ResourceIdentifier id = null,
                                      systemData,
                                      tags,
                                      location,
-                                     skuName.ToString(),
+                                     skuName?.ToSerialString(),
```

Comment:
Since SearchSkuName is an enum, calling ToString gives the Pascal cased names. We want the camel cased names which is provided by ToSerialString. SearchSkuName is hidden and replaced with SearchServiceSkuName which also uses camel case names.

Reply:
/cc @mattgotteiner